packages/testlab/src/validate-api-spec.ts

```diff
@@ -3,13 +3,13 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-import {OpenApiSpec} from '@loopback/openapi-v3-types';
 const validator = require('swagger2openapi/validate.js');
 import {promisify} from 'util';
 
 const validateAsync = promisify(validator.validate);
 
-export async function validateApiSpec(spec: OpenApiSpec): Promise<void> {
+// tslint:disable-next-line:no-any
+export async function validateApiSpec(spec: any): Promise<void> {
```

Comment:
hmm, any reason for changing the type of `spec`?
Assume the validator works fine, either `OpenApiSpec` or `any` works, while `OpenApiSpec` can catch error when editing I think.